### PR TITLE
update sdk to v0.6.1

### PR DESCRIPTION
**Pull request by yevgenypats**
This also fix https://github.com/cloudquery/cq-provider-azure/issues/103

### CODE REVIEW

**Review by roneli — CHANGES_REQUESTED** [review]
Would update the migration workflow, basically it should run on PR and the base version should be the first version we had migrations, so v0.3.7 I think

**Comment by yevgenypats** [comment]
@roneli looks like it was already running I've just added this to run on PR. in anycase the migrations tests should improved after we improve https://github.com/cloudquery/cloudquery-issues/issues/238 so we can test it via standard go test and not from outside.

**Review by roneli — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,11 +5,22 @@ All notable changes to this provider will be documented in this file.
...
-###### SDK Version: 0.5.7
+### 🚀 Added
+### :gear: Changed
+### :spider: Fixed
+### 💥 Breaking Changes
+
+-->
+
+## [v0.3.9] - 2022-01-03
+###### SDK Version: 0.6.1
```
Comment (roneli) [inline CHANGELOG.md:19]: missing changed upgrade for SDK and link to SDK version

**Review by roneli — COMMENTED** [review]
some nitpick on changelog

**Pull request merged**